### ISSUE TITLE
Lutris takes extremely long time to parse very long string keys in registry

### ISSUE THREAD

**Issue by Hark0nnen** [lutris/lutris#1418]
If user.reg contains very long (we a talking megabytes long here) string key it takes a very long time to start anything in this wineprefix with lutris. With ~15MB key lutris was taking ~30 minutes. It happens before starting wine, in "preparing to start" phase.

P.S. If you ask how does this happens - Battletech ny HBS writes some analytics into string key in registry. And it only adds to it, never cleans.

**Comment by Hark0nnen** [comment]
[user.reg.zip](https://github.com/lutris/lutris/files/2684074/user.reg.zip)

**Comment by strycore** [comment]
awesome, thanks!

**Comment by sigmaSd** [comment]
It gets stuck here https://github.com/lutris/lutris/blob/next/lutris/util/wine/registry.py#L111, because of the len of the reg, not sure how do u want to handle this, anyhow here is a regex to extract that part just in case `rg --multiline '"analytics\.sav.*"=hex:[\w+|\W+]*?"'`, the last `"` is not part of the match but I'm not sure how to extract it, still sharpening my regex skills :p

**Comment by strycore** [comment]
Fix merged
